versions/3.1.1.md

```diff
@@ -69,7 +69,7 @@ An OpenAPI definition can then be used by documentation generation tools to disp
 ## Definitions
 
 ##### <a name="oasDocument"></a>OpenAPI Description
-A document (or set of documents) that describes an API. An OpenAPI Description (OAD) uses and conforms to the OpenAPI Specification, and MUST contain at least one [paths](#pathsObject) field, [components](#oasComponents) field, or [webhooks](#oasWebhooks) field.
+An OpenAPI description describes an API, and is composed of an [entry document](#documentStructure), and all its referenced documents. An OpenAPI Description (OAD) uses and conforms to the OpenAPI Specification, and MUST contain at least one [paths](#pathsObject) field, [components](#oasComponents) field, or [webhooks](#oasWebhooks) field.
```

Comment:
Nitpick: Capitalize "Description" in "OpenAPI Description".  You could then move the "(OAD)" up and just use OAD in the 2nd sentence.

Other than that, I think we just need to get some confirmation in a TDC meeting that folks are comfortable with "and all its referenced documents."  It seems fine to me, but people have Opinons on this sort of thing.